README.md

```diff
@@ -382,6 +383,27 @@ memory and speed conditions. The default values should not be increased, unless
 both the memory and speed requirements are met. These values of system memory
 are also recommended based on the port speed for rTorrent to reduce disk usage.
 
+### rTorrent disk space preallocation
+
+Preallocate disk space for contents of a torrent
+
+* `RT_PREALLOCATE_TYPE`: Sets the type of disk space preallocation to use.
```

Comment:
Can you put it directly under this section https://github.com/crazy-max/docker-rtorrent-rutorrent?tab=readme-ov-file#rtorrent and cross link to "rTorrent disk space preallocation" section?

Reply:
Done.